docs/yuidoc-p5-theme/assets/shader-gradient.frag

```diff
@@ -0,0 +1,19 @@
+// Code adopted from "Creating a Gradient Color in Fragment Shader"
+// by Bahadır on stackoverflow.com
+// https://stackoverflow.com/questions/47376499/creating-a-gradient-color-in-fragment-shader
+
+precision highp float; varying vec2 vPos;
+uniform float pos;
+
+void main() {
+
+  vec2 st = vPos.xy + vec2(0,pos);
```

Comment:
@stalgiag I agree, and yet the reason why I opted for the Mandel shader was exactly the familiarity of its reoccurrence throughout the reference manual. But the fact that is it could seem like a blackbox for users/beginners is undeniable. 

Perhaps I should change `pos` to `offset`. Then the shader could  coherently be extended so that:
vec2 st = vPos.xy + vec2(xOffset,yOffset);

color1 and color2 could perhaps be uniforms as well. That way the shader could potentially be reused with a certain level of modularity that is visually distinguishable:
- moves along X || moves along Y || moves on XY plane || static
- full-color control

this could allow two instances of this shader to properly reflect the effects of `shader() ` for this particular example/commit:
shader1: blue-orange moving vertically
shader2: red-green moving horizontally 

what do yo think?